terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,88 @@
+    for sec in data.aws_ssm_parameter.secret:
+      trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
+  }
+
+  service_secrets_arn_map = {
+    for sec in module.secrets.secrets:
+      trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
+  }
+
+  task_secrets = [
+    { "name": "COOKIE_SECRET", "valueFrom": "${local.secrets_arn_map.web-oauth2-cookie-secret}" },
+    { "name": "CHS_API_KEY", "valueFrom": "${local.service_secrets_arn_map.chs_api_key}" },
+    { "name": "CACHE_SERVER", "valueFrom": "${local.service_secrets_arn_map.cache_server}" },
+    { "name": "OAUTH2_CLIENT_ID", "valueFrom": "${local.service_secrets_arn_map.oauth2_client_id}" },  
+    { "name": "OAUTH2_CLIENT_SECRET", "valueFrom": "${local.service_secrets_arn_map.oauth2_client_secret}" },
+    { "name": "ACCOUNT_URL", "valueFrom": "${local.service_secrets_arn_map.account_url}" },
+    { "name": "INTERNAL_API_URL", "valueFrom": "${local.service_secrets_arn_map.internal_api_url}" },
+    { "name": "PAYMENTS_API_URL", "valueFrom": "${local.service_secrets_arn_map.payments_api_url}" }
+  ]
+
+  task_environment = [
+    { "name": "NODE_PORT", "value": "${local.container_port}" },
+    { "name": "LOG_LEVEL", "value": "${var.log_level}" },
+    { "name": "CHS_URL", "value": "${var.chs_url}" },
+    { "name": "PIWIK_URL", "value": "${var.piwik_url}" },
+    { "name": "PIWIK_SITE_ID", "value": "${var.piwik_site_id}" },
+    { "name": "CDN_HOST", "value": "//${var.cdn_host}" },
+    { "name": "COOKIE_DOMAIN", "value": "${var.cookie_domain}" },
+    { "name": "COOKIE_NAME", "value": "${var.cookie_name}" },
+    { "name": "COOKIE_SECURE_ONLY", "value": "${var.cookie_secure_only}" },
+    { "name": "DEFAULT_SESSION_EXPIRATION", "value": "${var.default_session_expiration}" }, # TODO Is this needed?
+    { "name": "PIWIK_START_GOAL_ID", "value": "${var.piwik_start_goal_id}" },
+    { "name": "PIWIK_UPDATE_START_GOAL_ID", "value": "${var.piwik_update_start_goal_id}" },
+    { "name": "SHOW_SERVICE_OFFLINE_PAGE", "value": "${var.show_service_offline_page}" },
+    { "name": "FEATURE_FLAG_ENABLE_UPDATE_STATEMENT_VALIDATION_05072023", "value": "${var.feature_flag_enable_update_statement_validation_05072023}" },
+    { "name": "FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022", "value": "${var.feature_flag_enable_save_and_resume_17102022}" },
+    { "name": "FEATURE_FLAG_ENABLE_ROE_UPDATE_24112022", "value": "${var.feature_flag_enable_roe_update_24112022}" },
+    { "name": "FEATURE_FLAG_ENABLE_ROE_REMOVE_24112022", "value": "${var.feature_flag_enable_roe_remove_24112022}" },
+    { "name": "FEATURE_FLAG_ENABLE_TRUSTS_WEB_07112022", "value": "${var.feature_flag_enable_trusts_web_07112022}" },
+    { "name": "FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME_07032023", "value": "${var.feature_flag_enable_update_save_and_resume_07032023}" },
+    { "name": "FEATURE_FLAG_ENABLE_UPDATE_TRUSTS_30062023", "value": "${var.feature_flag_enable_update_trusts_30062023}" },
```

Comment:
{ "name": "FEATURE_FLAG_ENABLE_UPDATE_MANAGE_TRUSTS_29082023", "value": "${var. feature_flag_enable_update_manage_trusts_29082023}" },

Reply:
{ "name": "FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH_2806202", "value": "${var. feature_flag_disable_update_private_data_fetch_2806202}" },

Reply:
Yep, they are in the two config PRs I have done for CiDev and LIVE.